pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/Namespaces.java

```diff
@@ -571,7 +571,7 @@ public void splitNamespaceBundle(
             @QueryParam("authoritative") @DefaultValue("false") boolean authoritative,
             @QueryParam("unload") @DefaultValue("false") boolean unload,
             @QueryParam("splitAlgorithmName") String splitAlgorithmName,
-            @ApiParam("splitBoundaries") List<Long> splitBoundaries) {
+            @QueryParam("splitBoundaries") List<Long> splitBoundaries) {
```

Comment:
Should we also add the `@DefaultValue` here as the V1 API did?

Reply:
Good catch!

Reply:
Good catch!